shared/config/defaults.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+const (
+	DefaultRPCPort      = "8081"
+	DefaultRemoteCLIURL = "http://localhost:8081"
```

Comment:
RemoteCLIURL should correspond with the rpc port

Reply:
In other words, let's make the default port and host constants, and initialise the full URL in `init` use `fmt.Sprintf`